test(output): update tests to use libtest.h (fix #197)

jduo opened the pull request:
Change tests to use libtest.h as specified in
Issue #197.

Fixes #197 .

Changes proposed in this pull request:
- Use methods defined in libtest.h to improve test output formatting.

@lc-soft


<!-- Issuehunt content -->

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#197: Update test program](https://issuehunt.io/repos/5293802/issues/197)
---
</details>
<!-- /Issuehunt content-->

lc-soft commented on test/test.c:
```diff
@@ -11,20 +11,19 @@ int main(void)
+	describe("test object", test_object);
+	describe("test thread", test_thread);
+	describe("test font_load", test_font_load);
+	describe("test image_reader", test_image_reader);
```
Underscores should be replaced with spaces

```diff
- font_load
+ font load
```

jduo replied:
Done. Verified all describe() calls.

lc-soft commented on test/test_font_load.c:
```diff
@@ -4,153 +4,139 @@
-		"check get font (style: normal, weight: normal)",
-		id = GetSegoeUIFont( FONT_STYLE_NORMAL, FONT_WEIGHT_NORMAL )
-	);
+	it_i("Load segoeui.ttf", LCUIFont_LoadFile( "C:/windows/fonts/segoeui.ttf" ), 0 );
```
Please use `clang-format` to format the code

jduo replied:
I've used clang-format on all affected c files.

lc-soft commented:
> Note that TEST_LOG is still used in a few places
> where we log purely informational messages that
> don’t match functions in libtest.h

For code that uses TEST_LOG like below, , you can replace `TEST_LOG` with `Logger_Debug`.

https://github.com/lc-soft/LCUI/blob/4e6e459cd21f5281d13848a0cc80189e1c145618/test/test_thread.c#L19-L38

jduo commented:
> > Note that TEST_LOG is still used in a few places
> > where we log purely informational messages that
> > don’t match functions in libtest.h
> 
> For code that uses TEST_LOG like below, you can replace `TEST_LOG` with `Logger_Debug`.
> 
> https://github.com/lc-soft/LCUI/blob/4e6e459cd21f5281d13848a0cc80189e1c145618/test/test_thread.c#L19-L38

I've replaced calls to TEST_LOG with Logger_Debug. I've also deleted the now unused macros in test.h

jduo commented:
> The output text style should be consistent, I suggest you change them to lower case.
> 
> ```diff
> - Check *****
> + check *****
> ```
> 
> ![image](https://user-images.githubusercontent.com/1730073/84849487-fb374b80-b087-11ea-9fc7-a59ab74a37e7.png)
Done.
